install_config/persistent_storage/persistent_storage_aws.adoc

```diff
@@ -142,3 +142,12 @@ formatted with the given file system.
 
 This allows using unformatted AWS volumes as persistent volumes, because
 {product-title} formats them before the first use.
+
+=== Maximum number of EBS volumes on a node
+By default {product-title} supports maximum of 39 EBS volumes attachable to one Node. This limit is
```

Comment:
Change 'supports maximum of 39 EBS volumes attachable to one Node' to 'supports a maximum of 39 EBS volumes attached to one node'